android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```

Comment:
It probably doesn't matter but I found that Flutter is stil using `3.5.0` be default.

Reply:
Did it result in any warning?

Reply:
No, I think it's good.